frontend/src/routes.js

```diff
@@ -23,13 +23,13 @@ const routes = [
   },
   {
     exact: true,
-    path: '/dashboard/classes/:classId',
+    path: '/dashboard/classes/:classId/:tab',
     Component: Class,
     name: 'Klas',
   },
   {
     exact: true,
-    path: '/dashboard/classes/:classId/lessons/:lessonId',
+    path: '/dashboard/classes/:classId/lessons/:lessonId/:tab',
     Component: Lesson,
     name: 'Les',
```

Comment:
Deze PR zorgt ervoor dat de breadcrumbs een beetje kapot gaan. Als je gaat naar een les van een klas, krijg je niet meer `Klas/Les` te zien, maar alleen `Les`

Reply:
should be fixed now